README.md

```diff
@@ -28,7 +33,7 @@ What is the purpose of the `alt` attribute? e.g:
 ```
 
 ```text
-Your answer here.
+alt adds a description of the image.  So site visitors can use alternative, non-browser devices and still understand the content.
 ```
 
```

Comment:
While this is true, to be more specific- if the user is unable to view the image for whatever reason, the alt attribute will display the text of the image to indicate it's content. OR, if a user is blind or visually impaired and requires the use of a screen reader.